cinema/serializers.py

```diff
@@ -68,13 +79,77 @@ class Meta:
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available",
         )
 
 
+class TicketInfoSeatsSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Ticket
+        fields = ("row", "seat")
+        read_only_fields = fields
```

Comment:
The `read_only_fields` attribute is redundant here because all fields in `TicketInfoSeatsSerializer` are already marked as read-only by default. You can remove this line to clean up the code.